File: trail1/정수 직사각형/rectangle-of-numbers.c
#include <memory.h>
#include <stdio.h>

int main() {
    int n,m;
    int num = 1;

    scanf("%d ", &n);
    scanf("%d ", &m);

    int arr[n][m];
    memset(arr, 0, n * m * sizeof(int));

    for(int i = 0; i < n; i++){
        for(int j = 0; j < m; j++){
            arr[i][j] = num;
            num++;
        }
    }

    for(int i = 0; i < n; i++){
        for(int j = 0; j < m; j++){
            printf("%d ", arr[i][j]);
        }
        printf("\n");
    }
    
    return 0;
}

File: trail1/정수 직사각형/rectangle-of-numbers.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n,m;
        int num = 1;

        n = sc.nextInt();
        m = sc.nextInt();

        int[][] arr = new int[n][m];

        for(int i = 0; i < n; i++){
            for(int j = 0; j < m; j++){
                arr[i][j] = num;
                num++;
            }
        }

        for(int i = 0; i < n; i++){
            for(int j = 0; j < m; j++){
                System.out.printf("%d ", arr[i][j]);
            }
            System.out.println();
        }
    }
}